### DISCUSSION TITLE
Handling Retryable error for OTLP exporter

### DISCUSSION THREAD

**Discussion by mks26** [open-telemetry/opentelemetry-cpp#2042] [General]
Hi, 

If OTel collector is down or  due to some other reason if OTLP exporter received Retryable error code, are we doing any retry operation in opentelemetry-cpp sdk. I didn't find any code that does retry for OTLP exporters (both OTLP/GRPC and OTLP/HTTP).

Error codes that are retryable and non-retryable is mentioned in  OTLP  spec at :   
    https://opentelemetry.io/docs/reference/specification/protocol/otlp/

  OTLP/GRPC:    https://opentelemetry.io/docs/reference/specification/protocol/otlp/#failures
   OTLP/HTTP :   https://opentelemetry.io/docs/reference/specification/protocol/otlp/#failures-1

If it is not implemented yet, this can be a candidate for feature.

Thanks,
Mukesh

**Comment by lalitb** [comment]
Yes, retry is not supported. Please feel free to create separate GitHub issues for OTLP/gRPC and OTLP/HTTP.

**Reply by mks26** [reply]
Thanks @lalitb ! Created issue : https://github.com/open-telemetry/opentelemetry-cpp/issues/2049  for it.
